bindings/rust/bench/certs/ec384/server-cert.pem

```diff
@@ -0,0 +1,19 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Is there any reason why you need to commit these certificates to the repo? Maybe you could just have the criteria to run the generate_certs script before running the benchmarks?

Reply:
I was following how s2n-tls and s2n-quic have the certs checked in for testing. Because benching is also testing, having the certs checked in would make results more reproducible and remove another factor we have to think about.

Reply:
The reason why s2n-tls and s2n-quic have certificates committed is because we need them to exist for our CI runs. Your benchmarking tests don't run in our CI, so I am not really seeing a need to have them committed in the repo besides convenience. 
I dunno, it just seems weird to commit this many certs to our repo that we don't technically need.

Reply:
That makes sense that those certs I was looking at are for CI. Eventually I think the benching stuff will make it into CI (hopefully), but for now I could take them out.